### PR TITLE
Release 1.26.0

### PR DESCRIPTION
**Pull request by scrosby**
## Changes proposed in this PR

- Release
- 
-

### CODE REVIEW

File: scheduler/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file
  
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.26.0] - 2018-11-08
+### Added
+- Support for job launch rate limits, from @scrosby
+- Pool info to usage command for CLI
+- Quota to cs usage CLI command.
```
Comment (dposada) [inline scheduler/CHANGELOG.md]: Historically, we don't put CLI changes in this file

File: scheduler/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file
  
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.26.0] - 2018-11-08
+### Added
+- Support for job launch rate limits, from @scrosby
```
Comment (dposada) [inline scheduler/CHANGELOG.md]: Wasn't this part of `1.25.0`?

File: scheduler/CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file
...
+- Pool info to usage command for CLI
+- Quota to cs usage CLI command.
+- Global launch rate limit
```
Comment (dposada) [inline scheduler/CHANGELOG.md]:
```suggestion
- Global launch rate limit, from @scrosby 
```

File: scheduler/CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file
...
+### Changed
+- Data Locality service changed default port from 5000 to 35847
```
Comment (dposada) [inline scheduler/CHANGELOG.md]: Wasn't this only a local-dev facing change?

File: scheduler/CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file
...
+- Improved Docker build for integration tests
```
Comment (dposada) [inline scheduler/CHANGELOG.md]: Historically, we don't put integration test changes in this file

File: scheduler/CHANGELOG.md [hunk 6]
```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file
...
+- Made per-user rate limit more gradual.
```
Comment (dposada) [inline scheduler/CHANGELOG.md]:
```suggestion
- Made per-user rate limit more gradual, from @scrosby 
```

File: scheduler/CHANGELOG.md [hunk 7]
```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file
...
+- Sped up /unscheduled endpoint with new query and truncating long lists.
```
Comment (dposada) [inline scheduler/CHANGELOG.md]:
```suggestion
- Sped up `/unscheduled` endpoint with new query and truncating long lists, from @pschorf 
```

File: scheduler/CHANGELOG.md (hunk 2 above)
Comment (scrosby) [reply scheduler/CHANGELOG.md]: Merged with gradual rate limit below.

File: scheduler/CHANGELOG.md (hunk 3 above)
Comment (scrosby) [reply scheduler/CHANGELOG.md]: Done.

File: scheduler/CHANGELOG.md (hunk 1 above)
Comment (scrosby) [reply scheduler/CHANGELOG.md]: Done.

File: scheduler/CHANGELOG.md (hunk 4 above)
Comment (scrosby) [reply scheduler/CHANGELOG.md]: Yes.

File: scheduler/CHANGELOG.md (hunk 5 above)
Comment (scrosby) [reply scheduler/CHANGELOG.md]: Removed.

File: scheduler/CHANGELOG.md (hunk 6 above)
Comment (scrosby) [reply scheduler/CHANGELOG.md]: Done.

File: scheduler/CHANGELOG.md (hunk 7 above)
Comment (scrosby) [reply scheduler/CHANGELOG.md]: Changed.

File: scheduler/CHANGELOG.md [hunk 8]
```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file
  
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.26.0] - 2018-11-08
```
Comment (dposada) [inline scheduler/CHANGELOG.md]:
```suggestion
## [1.26.0] - 2018-12-11
```

**Pull request merged**